errbot/plugin_manager.py

```diff
@@ -465,11 +465,10 @@ def update_dynamic_plugins(self):
 
     def activate_non_started_plugins(self):
         """
-
+        Activates all plugins that are not activated, respecting its dependencies.
         :return: Empty string if no problem occured or a string explaining what went wrong.
```

Comment:
If you add a blank line in between the description and `:return:`, Sphinx will render this properly (it treats it as a single line otherwise and won't see `:return:` as a special tag then)